### PR TITLE
F#10 trapezoid table models

### LINKED ISSUE #10: Upload models of the trapezoid table
Upload blender models of  trapezoid table into robert_gazebo/cad/ directory

Upload the visual models of  trapezoid table in .dae extension into robert_gazebo/meshes/ directory

Upload the collision models of  trapezoid table in .dae extension into robert_gazebo/meshes/ directory

Upload models of  trapezoid table in .sdf extension into robert_gazebo/models/trapezoid table/ directory

Upload models of  trapezoid table in .config extension into robert_gazebo/models/trapezoid table/ directory

### PR DESCRIPTION
**Pull request by LAnastasiia**
closes #10

### CODE REVIEW

File: robert_gazebo/worlds/ucu.world [hunk 1]
```diff
@@ -10,6 +10,18 @@
       <uri>model://sun</uri>
     </include>
 
+    <include>
```
Comment (AndriyPt) [inline robert_gazebo/worlds/ucu.world]: please remove debug code

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Please remove blend1 files and not used STL files

File: robert_gazebo/worlds/ucu.world (hunk 1 above)
Comment (LAnastasiia) [reply robert_gazebo/worlds/ucu.world]: Done

**Comment by AndriyPt** [comment]
Bad scale of the table
image

**Comment by LAnastasiia** [comment]
Done.

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Please remove files with extension blend1 and stil

**Comment by LAnastasiia** [comment]
Done.

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**